packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -313,6 +310,57 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
   }
 }
 
+/// An picker default magnifier, grey rrect first introduced in iOS 14
+class CupertinoPickerDefaultMagnifier extends StatelessWidget {
+
+  /// Create a magnifier, it can be easily configured with the default style.
```

Comment:
Don't editorialize or use subjective adjectives. Just state what it does.

Reply:
The api doc has been updated and has a more detailed description of the arguments.